cmd/virt-handler/virt-handler.go

```diff
@@ -389,6 +389,21 @@ func (app *virtHandlerApp) Run() {
 		panic(err)
 	}
 
+	// Bootstrapping. From here on the startup order matters
+
+	factory.Start(stop)
```

Comment:
Moving this code up to ensure that the cache is synced before creating the VirtualMachineController, so that it can get the correct HypervisorConfiguration from the KV CR.